git-https-proxy/main.go

```diff
@@ -0,0 +1,190 @@
+package main
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/elazarl/goproxy"
+)
+
+func main() {
+	config := parseEnv()
+	proxyHandler := getProxyHandler(config)
+	proxyServer := http.Server{
+		Addr:    fmt.Sprintf(":%s", config.ProxyPort),
```

Comment:
i'm wondering why not "0.0.0.0:%s" ?

Reply:
That is a good point to make this explicit. I think that should work.

Reply:
Did it in both places.

If someone is here and wondering why not `127.0.0.1` it is because the k8s probes do not originate from localhost. But from another IP. So if I listen only on `127.0.0.1` then the probes fail.

And the probes are setup to go through the proxy to hit the health endpoint. So the proxy definitely cannot listen only on `127.0.0.1`